src/components/htmlvideoplayer/plugin.js

```diff
@@ -1069,6 +1100,14 @@ define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackMa
             });
         }
 
+        function renderSsaAss(videoElement, track, item) {
+            if (supportsCanvas() && supportsWebWorkers()) {
+                renderWithSubtitlesOctopus(videoElement, track, item);
+            } else {
+                renderWithLibjass(videoElement, track, item);
```

Comment:
can we have a `console.log()` about falling back to libjass?

Reply:
there is also a new conflict if you have time to fix it